internal/processor/filterset/strict/strictfilterset.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strict
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector/internal/processor/filterset"
+)
+
+// strictFilterSet encapsulates a set of exact string match filters.
+type strictFilterSet struct {
+	filters map[string]bool
```

Comment:
nit: since the actual value doesn't matter but only the presence of the key I have a slight preference to use the empty `struct{}` instead.

Reply:
Thanks for the review! I took both your suggestions. I actually didn't know about that empty struct{} was 0 bytes before, so that was cool to learn.